kpi/serializers/v2/service_usage.py

```diff
@@ -211,36 +213,37 @@ def _get_organization_details(self):
             return
 
         organization = Organization.objects.filter(
-            owner__organization_user__user=self.context.get('request').user,
+            organization_users__user=self.context.get('request').user,
             id=organization_id,
         ).first()
 
         if not organization:
             # Couldn't find organization, proceed as normal
             return
 
-        """
-        Commented out until the Enterprise plan is implemented
-
-        # If the user is in an organization, get all org users so we can query their total org usage
-        self._user_ids = list(
-            User.objects.values_list('pk', flat=True).filter(
-                organizations_organization__id=organization_id
-            )
-        )
-        """
-
         # If they have a subscription, use its start date to calculate beginning of current month/year's usage
-        billing_details = organization.active_subscription_billing_details
-        if billing_details:
+        if billing_details := organization.active_subscription_billing_details:
             self._anchor_date = billing_details['billing_cycle_anchor'].date()
             self._period_start = billing_details['current_period_start'].date()
             self._period_end = billing_details['current_period_end'].date()
             self._subscription_interval = billing_details['recurring_interval']
 
+        if settings.STRIPE_ENABLED:
+            # if the user is in an organization and has an enterprise plan, get all org users
+            self._user_ids = list(
```

Comment:
As a reader of the code, I'd appreciate an explanation for making the otherwise lazy queryset into a executed list here. I could speculate on why - but that merits a request for a comment.